metadata-ingestion/src/datahub/utilities/mapping.py

```diff
@@ -299,6 +311,15 @@ def convert_to_aspects(self, operation_map: Dict[str, list]) -> Dict[str, Any]:
             )
             aspect_map[Constants.ADD_TERM_OPERATION] = term_aspect
 
+        if Constants.ADD_DOMAIN_OPERATION in operation_map:
+            domain_aspect = DomainsClass(
+                domains=[
```

Comment:
Don't we want to check if domain exists?
We do this in our SQL sources -> https://github.com/datahub-project/datahub/blob/56bb4c81113bf4fce8a48526080375480d619ba8/metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py#L356

Reply:
in the SQL sources we try to map name -> domain. here, we only support domain urns so don't need the DomainRegistry